sharding/indexHashedNodesCoordinator.go

```diff
@@ -79,11 +104,28 @@ func (ihgs *indexHashedGroupSelector) ComputeValidatorsGroup(randomness []byte)
 	return tempList, nil
 }
 
+// GetSelectedValidatorsPublicKeys calculates the validators group for a specific randomness,
```

Comment:
Comment does not start with function name.

Reply:
done